src/labbook/components/thumbnail-chooser/thumbnail-chooser.test.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+import { IThumbnailChooserProps, ThumbnailChooser } from "./thumbnail-chooser";
+
+describe("ThumbnailChooser component", () => {
+  it("renders", () => {
+
+    const Thumbnail = () => <div data-testid="thumbnail">Thumbnail</div>;
+
+    const thumbnailChooserProps: IThumbnailChooserProps = {
+      items: [{
+        id: "a",
+        empty: false,
+        data: {}
+      }],
+      RenderingF: Thumbnail,
+      selectedItemId: null,
+      setSelectedItemId: (id: string) => null,
+      clearSelectedItemId: (id: string) => null,
+      maxDisplayItems: 4
+    };
+
+    render(<ThumbnailChooser {...thumbnailChooserProps} />);
+    expect(screen.getAllByTestId("thumbnail-chooser")).toHaveLength(1);
+    expect(screen.getAllByTestId("thumbnail")).toHaveLength(1);
```

Comment:
These tests can be more conventionally written as:
```typescript
    expect(screen.getByTestId("thumbnail")).toBeInTheDocument();
```
`getByTestId()` will throw an exception if there's zero or more than one.